sdk/keyvault/perf-tests/keyvault-keys/test/cryptography/decrypt.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { RsaEncryptionAlgorithm } from "@azure/keyvault-keys";
+import { randomBytes } from "crypto";
+import { CryptographyTest } from "./cryptography.spec";
+
+export class DecryptTest extends CryptographyTest {
+  public options = {};
+  private cipherText?: Uint8Array;
+  private algorithm: RsaEncryptionAlgorithm = "RSA-OAEP-256";
+
+  async setup(): Promise<void> {
+    const encryptResult = await CryptographyTest.cryptoClient!.encrypt({
+      algorithm: this.algorithm,
+      plaintext: randomBytes(32)
+    });
+    this.cipherText = encryptResult.result!;
```

Comment:
that will probably not pass the linting check. Please make sure `rushx lint` succeeds.